legal-api/src/legal_api/resources/v2/configuration.py

```diff
@@ -35,3 +36,60 @@ def get_configurations():
             configuration.json for configuration in configurations
         ]
     }), HTTPStatus.OK
+
+
+@bp.route('', methods=['PUT'])
+@cross_origin(origin='*')
+# @jwt.has_one_of_roles([UserRoles.staff])
```

Comment:
Just a reminder to uncomment it before it can be merged. 👀

Reply:
updated